src/hct_mis_api/apps/household/forms.py

```diff
@@ -148,9 +149,7 @@ class MassRestoreForm(RestoreForm):
 class AddToTargetPopulationForm(forms.Form):
```

Comment:
AddToTargetPopulationForm/CreateTargetPopulationForm shouldn't be removed?

Reply:
removed